server/routerlicious/packages/services-utils/src/redisClientConnectionManager.ts

```diff
@@ -99,6 +99,10 @@ export class RedisClientConnectionManager implements IRedisClientConnectionManag
 	}
 
 	private authenticateAndCreateRedisClient(): void {
+		if (this.enableClustering) {
+			this.redisOptions.clusterRetryStrategy = this.redisOptions.retryStrategy;
```

Comment:
Can you please confirm if the values for the retry strategy are configurable?

Reply:
Added a commit to make this configurable